workflows/multiomics/full_pipeline/main.nf

```diff
@@ -7,6 +7,7 @@ include { split_modalities } from targetDir + '/dataflow/split_modalities/main.n
 include { merge } from targetDir + '/dataflow/merge/main.nf'
 include { concat } from targetDir + '/dataflow/concat/main.nf'
 include { remove_modality }  from targetDir + '/filter/remove_modality/main.nf'
+include { publish }  from targetDir + '/transfer/publish/main.nf'
```

Comment:
The publish component has returned! :fearful:

Reply:
Yes, I was thinking about ways to avoid it,  but there is currently no way to really know what the last step of the pipeline is because we use `reduce` to chain the integration setup step for each modality. The last step is based on whether or not a modality is present and the order in which they are processed in the reduce. This becomes even more of a headache when considering samples that only have 'unprocessed' modalities and no integration setup is applied (this is an edge-case, but still...). So I found using publish was the easiest solution :)